### PR TITLE
refactor: Make code style better

### PR DESCRIPTION
**Pull request by luckyJeffy**
May make the code more readable 😆

### CODE REVIEW

File: src/views/TradingCompetition/index.tsx [hunk 1]
```diff
@@ -99,8 +99,11 @@ const TradingCompetition = () => {
   const [team3LeaderboardInformation, setTeam3LeaderboardInformation] = useState({ teamId: 3, leaderboardData: null })
 
   const isCompetitionLive = currentPhase.state === LIVE
-  const hasCompetitionEnded =
-    currentPhase.state === FINISHED || currentPhase.state === CLAIM || currentPhase.state === OVER
+  const hasCompetitionEnded = [
+    FINISHED,
+    CLAIM,
+    OVER
+  ].some((phasesItem) => phasesItem === currentPhase.state)
```
Comment (hachiojidev) [inline src/views/TradingCompetition/index.tsx:106]: In this case I think it is better to use `includes`
```
const hasCompetitionEnded = [
  FINISHED,
  CLAIM,
  OVER
].includes(currentPhase.state)
```

File: src/views/TradingCompetition/index.tsx [hunk 2]
```diff
@@ -201,8 +204,12 @@ const TradingCompetition = () => {
...
 
   // Don't hide when loading. Hide if the account is connected && the user hasn't registered && the competition is live or finished
-  const shouldHideCta =
-    !isLoading && account && !userTradingInformation.hasRegistered && (isCompetitionLive || hasCompetitionEnded)
+  const shouldHideCta = [
+    !isLoading,
+    account,
+    !userTradingInformation.hasRegistered,
+    isCompetitionLive || hasCompetitionEnded
+  ].every(i => Boolean(i))
```
Comment (hachiojidev) [inline src/views/TradingCompetition/index.tsx:212]: I am not convinced this is more _readable_. I would opt for an inline function.
```
const shouldHideCta = () => {
  if (!isLoading) {
    return false
  }  

// Guard for each item  
...etc
  
  return true
}
```

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: src/views/TradingCompetition/index.tsx (hunk 1 above)
Comment (luckyJeffy) [reply src/views/TradingCompetition/index.tsx:106]: Yeah, It was much better 👍

File: src/views/TradingCompetition/index.tsx (hunk 2 above)
Comment (luckyJeffy) [reply src/views/TradingCompetition/index.tsx:212]: I got it, this case can be more controllable.
But now the constance "shouldHideCta" type will become to Function.

File: src/views/TradingCompetition/index.tsx (hunk 1 above)
Comment (luckyJeffy) [reply src/views/TradingCompetition/index.tsx]: `
export const EndedPhasesSet = new Set([
  FINISHED,
  CLAIM,
  OVER
])
`

In this case, Set may better than Array.
So I just try modify the constant file for a new set, then import by "TradingCompetition" to use it.

d68109852c96cc46d309396f0e54c258c9274505

File: src/views/TradingCompetition/index.tsx (hunk 2 above)
Comment (luckyJeffy) [reply src/views/TradingCompetition/index.tsx]: Updated.

d68109852c96cc46d309396f0e54c258c9274505

File: src/views/TradingCompetition/index.tsx [hunk 3]
```diff
@@ -99,9 +98,7 @@ const TradingCompetition = () => {
   const [team3LeaderboardInformation, setTeam3LeaderboardInformation] = useState({ teamId: 3, leaderboardData: null })
 
   const isCompetitionLive = currentPhase.state === LIVE
-  const hasCompetitionEnded =
-    currentPhase.state === FINISHED || currentPhase.state === CLAIM || currentPhase.state === OVER
-
+  const hasCompetitionEnded = EndedPhasesSet.has(currentPhase.state);
```
Comment (hachiojidev) [inline src/views/TradingCompetition/index.tsx:101]: Nice. Have not considered this pattern but it is definitely more readable.

**Review by hachiojidev — APPROVED** [review]

File: src/views/TradingCompetition/index.tsx (hunk 3 above)
Comment (luckyJeffy) [reply src/views/TradingCompetition/index.tsx:101]: 🎉

**Comment by luckyJeffy** [comment]
`
Run yarn format:check
yarn run v1.22.10
$ prettier --check --loglevel error 'src/**/*.{js,jsx,ts,tsx}'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
`

Seam like something went wrong, I got same issue here when I try to commit the code.

**Comment by Chef-Chungus** [comment]
@luckyJeffy - Try doing `yarn install` then running prettier on the files again.

**Comment by hachiojidev** [comment]
@luckyJeffy 

>Seam like something went wrong, I got same issue here when I try to commit the code.

run `yarn format:write`

Likely there is a file that was not formatted. If it is a file you have not touched try rebasing develop.

(site note: need to figure out why prettier is not reporting the warnings in the console)

**Comment by Chef-Chungus** [comment]
@luckyJeffy We will close this PR in one day if there's no update from you

**Comment by luckyJeffy** [comment]
> @luckyJeffy We will close this PR in one day if there's no update from you

Sorry guys, I am going through a very busy stage (Maybe two weeks), I will continue to update when I take a little break

**Pull request closed without merging**